README.md

```diff
@@ -94,6 +94,9 @@ A curated list of resources for learning about natural language processing, text
 ## Blog Articles, Papers, Case Studies
 
 ### General <a id="general-articles"></a>
+- [Why Accuracy in Natural Language Processing is Crucial to the Future of AI in Retail](https://www.mytotalretail.com/article/why-accuracy-in-natural-language-processing-is-crucial-to-the-future-of-ai-in-retail/)
+- [Natural Language Processing for Information Extraction](https://arxiv.org/abs/1807.02383)
+- [Natural Language Processing (NLP) Techniques for Extracting Information](https://www.searchtechnologies.com/blog/natural-language-processing-techniques) 
```

Comment:
This link, and the one above, should be in the Entity and Information Extraction section.